inductiva/tasks/task.py

```diff
@@ -215,8 +218,14 @@ def get_output(
             100%|██████████| 1.64G/1.64G [00:32<00:00, 55.1MiB/s]
         """
         self.wait()
+
+        if full_output:
+            filenames = []
+        else:
+            filenames = self._default_files_list
```

Comment:
`filenames = self._default_files_list if full else []`

Reply:
Actually, shouldn't `filenames` be `None` when you want to download all files? Passing an empty list sounds like you want to download *none of the files*.